lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -0,0 +1,101 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'js/financial-assistance/models/financial_assistance_model',
+            'text!js/financial-assistance/templates/financial_assessment_form.underscore',
+            'text!js/financial-assistance/templates/financial_assessment_submitted.underscore',
+            'js/student_account/views/FormView',
+            'text!templates/student_account/form_field.underscore'
+         ],
+         function(Backbone, $, _, gettext, FinancialAssistanceModel, formViewTpl, successTpl, FormView, formFieldTpl ) {
```

Comment:
A few formatting nitpicks in this file -- the parameters here have a space on the right but not the left.